inst/shinyexample/R/run_app.R

```diff
@@ -1,11 +1,23 @@
 #' Run the Shiny Application
 #'
+#' @param onStart optional, described in ?shiny::shinyApp
```

Comment:
Can you use `@inheritParams shiny::shinyApp` here ?

Reply:
Fixed!
I was unaware of `@inheritParams `. That is going to save me so much time in the future.